// tapable/lib/index.js
/*
	MIT License http://www.opensource.org/licenses/mit-license.php
	Author Tobias Koppers @sokra
*/
"use strict";

module.exports.AsyncParallelBailHook = require("tapable/lib/AsyncParallelBailHook");
module.exports.AsyncParallelHook = require("tapable/lib/AsyncParallelHook");
module.exports.AsyncSeriesBailHook = require("tapable/lib/AsyncSeriesBailHook");
module.exports.AsyncSeriesHook = require("tapable/lib/AsyncSeriesHook");
module.exports.AsyncSeriesLoopHook = require("tapable/lib/AsyncSeriesLoopHook");
module.exports.AsyncSeriesWaterfallHook = require("tapable/lib/AsyncSeriesWaterfallHook");
module.exports.HookMap = require("tapable/lib/HookMap");
module.exports.MultiHook = require("tapable/lib/MultiHook");
module.exports.SyncBailHook = require("tapable/lib/SyncBailHook");
module.exports.SyncHook = require("tapable/lib/SyncHook");
module.exports.SyncLoopHook = require("tapable/lib/SyncLoopHook");
module.exports.SyncWaterfallHook = require("tapable/lib/SyncWaterfallHook");
module.exports.__esModule = true;
